dart/lib/src/sentry_envelope.dart

```diff
@@ -0,0 +1,25 @@
+import 'sentry_envelope_header.dart';
+import 'sentry_envelope_item.dart';
+import 'protocol/sentry_event.dart';
+import 'protocol/sdk_version.dart';
+
+class SentryEnvelope {
+  SentryEnvelope(this.header, this.items);
+
+  final SentryEnvelopeHeader header;
+  final List<SentryEnvelopeItem> items;
+
+  static SentryEnvelope fromEvent(SentryEvent event, SdkVersion sdkVersion) {
+    return SentryEnvelope(SentryEnvelopeHeader(event.eventId, sdkVersion),
+        [SentryEnvelopeItem.fromEvent(event)]);
+  }
```

Comment:
Dart has better alternatives than static functions for creating an instance of an object, could be a factory, named ctor or whatsoever

Reply:
Yeah, for example:
```dart
factory SentryEnvelope.fromEvent(SentryEvent event, SdkVersion sdkVersion) {
    return SentryEnvelope(SentryEnvelopeHeader(event.eventId, sdkVersion),
        [SentryEnvelopeItem.fromEvent(event)]);
}
```